packages/medusa/src/services/event-bus.ts

```diff
@@ -124,13 +148,26 @@ export default class EventBusService {
    * happens. Subscribers must return a Promise.
    * @return this
    */
-  subscribe(event: string | symbol, subscriber: Subscriber): this {
+  subscribe(
+    event: string | symbol,
+    subscriber: Subscriber,
+    context?: SubscriberContext
```

Comment:
self-review: I will need to document this.

Reply:
I suggest to add the jsdoc on the prop in the type wdyt?